addons/point_of_sale/static/src/js/devices.js

```diff
@@ -448,9 +448,11 @@ var ProxyDevice  = core.Class.extend(mixins.PropertiesMixin,{
 
     print_sale_details: function() {
         var self = this;
+        var config_id = [this.pos.config.id];
```

Comment:
config_id is only used inside the rpc query, so no need to make it a variable first.

Reply:
also true, but this is also preparing for future developments, where config_id could be an argument of the function.
Also, style-wise, I think i prefer to keep it as a separate variable

Reply:
> @switch87
> The problem here is less functional than technical.
> We all agree that the current feature is misleading, but it has this behavior since v10.0 (well, v9.0 actually). Furthermore, we agree on what the feature should become.
> Changing it will confuse even more the users. And I'll get tickets to revert this change.
> So, as the first rule of bugfixing is "Do no harm", I won't merge this PR, in any version.
> What should really be done, is a clarification/extension of the feature in master

Ok, I will refer to this pr for future development :+1: